src/main/java/org/openpnp/machine/reference/feeder/wizards/ReferenceTubeFeederConfigurationWizard.java

```diff
@@ -19,13 +19,13 @@
 
 package org.openpnp.machine.reference.feeder.wizards;
 
-import org.openpnp.machine.reference.feeder.ReferenceTubeFeeder;
+import org.openpnp.machine.reference.ReferenceFeeder;
 
 public class ReferenceTubeFeederConfigurationWizard
         extends AbstractReferenceFeederConfigurationWizard {
-    private final ReferenceTubeFeeder feeder;
+    private final ReferenceFeeder feeder;
```

Comment:
As TubeFeeder actually is the feeder with less features, basic feeder
wizard, it's used
for other feeders, this wizard could be used by other feeders that
need just minimal configurations.
Should i copy this as ReferenceFeederConfigurationWizard.java ? is this better ?

2016-07-26 17:28 GMT, Jason von Nieda <notifications@github.com>:
>>
>>  public class ReferenceTubeFeederConfigurationWizard
>>          extends AbstractReferenceFeederConfigurationWizard {
>> -    private final ReferenceTubeFeeder feeder;
>> +    private final ReferenceFeeder feeder;
>
> What is the purpose of this change? It limits access to any feeder specific
> properties and this is not how any of the other wizards work.
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/openpnp/openpnp/pull/305/files/15ab3f127e639a1e905fb7668367edd1280810a3#r72297218